test/unit/tasks/assets_test.rb

```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class AssetsRake < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  describe "assets:precompile" do
+    let(:task) { Rake::Task["assets:precompile"] }
+    let(:mustache) { Rake::Task["shared_mustache:compile"] }
+
+    test "shared mustache templates get compiled" do
+      mustache.reenable
+      task.reenable
+      mustache.expects(:execute)
+
+      task.invoke
+    end
+  end
+end
```

Comment:
I don't think there's much value in having this test since we're really just testing rake enhance works. And if it's not set-up we'd expect to have problems with tests.

Reply:
Removed it, thanks